README.md

```diff
@@ -1,2 +1,19 @@
 # r-image-creator
 Github Action to create R-based Docker images
+
+
+# action inputs description 
```

Comment:
Maybe could be generated by https://github.com/insightsengineering/actions-docgen ?